bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/converter/FibaroFGRM222Converter.java

```diff
@@ -0,0 +1,134 @@
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    openHAB.org - initial API and implementation and/or initial documentation
+ */
+package org.openhab.binding.zwave.internal.converter;
+
+import java.util.Map;
+
+import org.openhab.binding.zwave.internal.converter.command.MultiLevelPercentCommandConverter;
+import org.openhab.binding.zwave.internal.converter.command.MultiLevelUpDownCommandConverter;
+import org.openhab.binding.zwave.internal.converter.command.ZWaveCommandConverter;
+import org.openhab.binding.zwave.internal.converter.state.IntegerOnOffTypeConverter;
+import org.openhab.binding.zwave.internal.converter.state.IntegerPercentTypeConverter;
+import org.openhab.binding.zwave.internal.converter.state.ZWaveStateConverter;
+import org.openhab.binding.zwave.internal.protocol.SerialMessage;
+import org.openhab.binding.zwave.internal.protocol.ZWaveController;
+import org.openhab.binding.zwave.internal.protocol.ZWaveNode;
+import org.openhab.binding.zwave.internal.protocol.commandclass.proprietary.FibaroFGRM222CommandClass;
+import org.openhab.binding.zwave.internal.protocol.event.ZWaveCommandClassValueEvent;
+import org.openhab.core.events.EventPublisher;
+import org.openhab.core.items.Item;
+import org.openhab.core.library.types.DecimalType;
+import org.openhab.core.library.types.PercentType;
+import org.openhab.core.library.types.StopMoveType;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.State;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ *
+ * @author wenzel
+ * @author Markus Rathgeb <maggu2810@gmail.com>
+ */
+public class FibaroFGRM222Converter extends ZWaveCommandClassConverter<FibaroFGRM222CommandClass> {
+
+	private static final Logger logger = LoggerFactory.getLogger(FibaroFGRM222Converter.class);
+	private static final int REFRESH_INTERVAL = 0; // refresh interval in seconds
+
+	public FibaroFGRM222Converter(final ZWaveController controller, final EventPublisher eventPublisher) {
+		super(controller, eventPublisher);
+		this.addStateConverter(new IntegerPercentTypeConverter());
+		this.addStateConverter(new IntegerOnOffTypeConverter());
+
+		this.addCommandConverter(new MultiLevelPercentCommandConverter());
+		this.addCommandConverter(new MultiLevelUpDownCommandConverter());
+	}
+
+	@Override
+	SerialMessage executeRefresh(final ZWaveNode node, final FibaroFGRM222CommandClass commandClass, final int endpointId,
+			final Map<String, String> arguments) {
+		logger.debug("executeRefresh() -- nothing to do");
+		return null;
+	}
+
+	@Override
+	void handleEvent(final ZWaveCommandClassValueEvent event, final Item item, final Map<String, String> arguments) {
+		logger.debug("handleEvent()");
+		ZWaveStateConverter<?, ?> converter = this.getStateConverter(item, event.getValue());
+
+		if (converter == null) {
+			logger.warn("No converter found for item = {}, node = {} endpoint = {}, ignoring event.", item.getName(),
```

Comment:
Just for standardisation, can you make the debug message start with "NODE {}:" and remove the node={} later in the message.